src/app/components/Fragment/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Text from '../Text';
+
+const getFontVariant = (attr: string[]) => {
+  if (attr.includes('italic') && attr.includes('bold')) {
+    return 'sansBoldItalic';
+  }
+  if (attr.includes('bold')) {
+    return 'sansBold';
+  }
+  if (attr.includes('italic')) {
+    return 'sansRegularItalic';
+  }
+  return 'sansRegular';
+};
+
+const Fragment = ({ text, attributes = [] }: FragmentProps) => {
+  const fontVariant = getFontVariant(attributes);
+  return <Text fontVariant={fontVariant}>{text}</Text>;
```

Comment:
I think the `as` prop will need used here as well, as the old implementation rendered either an `<i>` or `<b>` tag:
```
const bold = ({ children }: PropsWithChildren) => <b>{children}</b>;

const italic = ({ children }: PropsWithChildren) => (
  <i css={styles.italicStyle}>{children}</i>
);
```

Looks like the `<i>` is styled whereas the `<b>` isn't too.